Compiler/src/abstractinterpretation.jl

```diff
@@ -2227,6 +2227,12 @@ function get_ci_mi(ci::CodeInstance)
     return def::MethodInstance
 end
 
+function get_ci_abi(ci::CodeInstance)
+    def = ci.def
+    isa(def, ABIOverride) && return def.abi
+    return (def::MethodInstance).specTypes
+end
+
```

Comment:
Seems to be a duplicated definition? We probably want to rename `ci_abi` to `get_ci_abi`.